geppetto.js/geppetto-ui/package.json

```diff
@@ -106,15 +106,23 @@
     "@babel/preset-env": "^7.8.4",
     "@babel/preset-react": "^7.8.3",
     "@babel/preset-typescript": "^7.13.0",
+    "@types/jasmine": "^3.7.2",
     "@types/jest": "^26.0.23",
     "@types/mocha": "^8.2.2",
     "@types/node": "^15.0.2",
+    "@types/react": "^17.0.5",
+    "@types/react-dom": "^17.0.4",
+    "awesome-typescript-loader": "^5.2.0",
     "babel-eslint": "^10.0.1",
     "babel-jest": "^25.1.0",
     "babel-loader": "^8.0.6",
     "babel-plugin-module-resolver": "^4.0.0",
     "eslint": "^6.0.1",
+    "jasmine": "^3.7.0",
     "jest": "^24.9.0",
+    "prettier": "^2.3.0",
+    "sass": "^1.32.12",
```

Comment:
why add sass?

Reply:
okay found out during investigation of a different issue :)
flex-layout defines scss styles